mypy/test/testcmdline.py

```diff
@@ -0,0 +1,77 @@
+"""Test cases for the command line.
+
+To begin we test that "mypy <directory>[/]" always recurses down the
+whole tree.
+"""
+
+import os
+import re
+import subprocess
+import sys
+
+from typing import Tuple, List, Dict, Set
+
+from mypy.myunit import Suite, SkipTestCaseException
+from mypy.test.config import test_data_prefix, test_temp_dir
+from mypy.test.data import parse_test_cases, DataDrivenTestCase
+from mypy.test.helpers import assert_string_arrays_equal
+
+# Path to Python 3 interpreter
+python3_path = sys.executable
+
+# Files containing test case descriptions.
+cmdline_files = ['cmdline.test']
+
+
+class PythonEvaluationSuite(Suite):
```

Comment:
Maybe we should modify `runtests.py` to not consider these unit tests, as running mypy as a subprocess is slow. See `add_myunit` and `add_pythoneval` in `runtests.py`. I suggest special casing these to be considered evaluation test cases (though the name is not accurate) or adding a new category.